src/AddTodoForm.js

```diff
@@ -1,21 +1,19 @@
-import react from "react";
-function AddTodoForm() {
+import React from "react";
+function AddTodoForm(props) {
     const handleAddTodo = (event) => {
         event.preventDefault();
-        setNewTodo(event.target.value);
-        console.log(event.target.value);
-        handleAddTodo.remove()
-       
-    };
-
+        const todoTitle = event.target.title.value;
+        props.setNewTodo(todoTitle);
+        event.target.reset();
+       };
     return (
-        <form>
-            <label htmlFor="todoTitle">Tittle</label>
-            <input type="text" id="todoTitle" name="title" onSubmit = {handleAddTodo } onAddTodo = {setNewTodo}></input>
-         <button>Add</button>
-             
+        <form onSubmit={handleAddTodo}>        <label htmlFor="todoTitle">Tittle</label>
+            <input type="text" id="todoTitle" name="title" onChange={handleAddTodo}></input>
+         
+            <button type="submit">Add</button>
+         
         </form>
```

Comment:
There's a few unnecessary empty lines here. Sometimes it helps to split large chunks of HTML up with empty lines, but generally you should avoid empty lines as it makes it more difficult to read.